content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md

```diff
@@ -0,0 +1,150 @@
+
+If you are new to Aruba Central and are looking to enable SSO, this guide is for you. This will walk you through the process of configuring SSO for GreenLake and Aruba Central using Okta.
+
+### Before you Begin
+
+Please review the [GreenLake](https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us) User Guide to understand how the SAML framework works in the context of Common Cloud Services for the Aruba Central application.
+
+### Steps to Configure SSO/SAML Application in Okta
+
+To configure application metadata in Okta, complete the following steps:
+
+* Step 1: Create an Okta SAML application
+* Step 2: Configure Sign On settings
+* Step 3: Export the SAML 2.0 IdP metadata
+* Step 4: Configure the SAML connection in GreenLake
+
+**Step 1: Create an Okta SAML Application**
+
+1. Log in to the Okta administration console.
+2. Click **Applications > Create App Integration.** The Create a new app integration window opens.
+3. Select SAML 2.0 and click Next.
+
+![](/img/ws-image0.png)
+
+Provide a name for the Aruba GreenLake SSO service (Okta Application)
+
+![](/img/ws-image1.png)
+
+**Step 2: Configure Single Sign On settings**
+
+1. Enter the SAML information.
+
+   Under General:
+
+    **Single Sign on URL:** https://sso.common.cloud.hpe.com/sp/ACS.saml2
+
+    **Audience URI (SP Entity ID):** https://sso.common.cloud.hpe.com
+
+    **Name ID format EmailAddress**
+
+    **Application username Email**
+
+    **NameID = user.email**
+
+    **gl\_first\_name = user.FirstName**
+
+    **gl\_last\_name = user.LastName**
+
+    **hpe\_ccs\_attribute = (See Below)**
+
+   See here for IdP attribute details: <https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us>
+
+   One of the additional features added as part of the GreenLake platform is Role Based Access Controls (RBAC) for Aruba Central and all other apps in GLCP. A new SAML attribute has been added “hpe_ccs_attribute” which tells GreenLake and Central the exact role/permissions for each user. The following describes how to format the attribute.
+
+![](/img/ws-image2.png)
+
+![](/img/ws-image3.png)
+
+![](/img/ws-image4.png)
+
+![](/img/ws-image5.png)
+
+The **hpe\_ccs\_attribute** will always start with **version_1#**. First, we’ll configure the attributes for GreenLake CCS, then Central. First enter the PCID for the account, followed by the GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Then followed by the role name and **ALL_SCOPES**. Next will be the Aruba Central info. Start with the **app cid**, then the role name (IE Aruba Central Administrator) then **ALL_SCOPES**.
```

Comment:
Updated w/ suggested edits:

The **hpe\_ccs\_attribute** always starts with version_1#. You must first configure the attributes for HPE GreenLake CSS, and then Central. To do so, enter the PCID for the account, followed by the HPE GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Following this, enter the role name and **ALL_SCOPES**. Next, enter in the Aruba Central information. Start with the **app cid**, followed by the role name (i.e. Aruba Central Administrator), and then **ALL_SCOPES**.